src/Properties/launchsettings.json

```diff
@@ -6,7 +6,7 @@
     "DevHome (Unpackaged)": {
       "commandName": "Project"
     },
-    "PI (Package)": {
+    "DevInsights (Package)": {
       "commandName": "MsixPackage",
       "commandLineArgs": "--utilitylaunch DevHome.PI.Exe ",

```

Comment:
Do you want to rename the exe?

Reply:
happening in phases, first phase is tackling the resw files. Then we'll rename the source/classes which would include the exe.